core/core.ex

```diff
@@ -48,7 +51,20 @@ defmodule AntikytheraCore do
     System.put_env("ERL_LIBS", new_value)
   end
 
-  defp establish_connections_to_other_nodes(tries_remaining \\ 3) do
+  @connection_retrial_interval_in_milliseconds 5_000
+
+  defunpt calculate_connection_trial_count_from_health_check_grace_period() :: pos_integer do
```

Comment:
- I renamed `calculate_connection_retry_count_from_health_check_grace_period/0` => `calculate_connection_trial_count_from_health_check_grace_period/0` in https://github.com/access-company/antikythera/pull/168/commits/746e621e2643650452f75bd628cfb0f93ea7df1d.
- This is because the return value is not "retrial count" but "trial count" considering the way to use it in `establish_connections_to_other_nodes/1`